### ISSUE TITLE
Handle IAsyncEnumerable

### ISSUE THREAD

**Issue by virzak** [lsoft/SyncToAsyncExtension#1]
Hi @lsoft,

It would be useful if the extension could also pick up siblings of IAsyncEnumerable. For example:

```cs
public static async IAsyncEnumerable<double> MovingAverageAsync(this IAsyncEnumerable<double> source, int windowSize, IProgress<int>? progress = null, [EnumeratorCancellation] CancellationToken ct = default)
```

to:

```cs
public static IEnumerable<double> MovingAverage(this IEnumerable<double> source, int windowSize)
```

Full source code is [here](https://github.com/zompinc/sync-method-generator/blob/5924db16483181a746dadc4ef2adff767d1a0c7c/tests/GenerationSandbox.Tests/AsyncExtensions.cs#L14-L59).

**Comment by lsoft** [comment]
@virzak Looks like I fixed it and published a new version 1.6 of the Vsix to the marketplace. Please let me know if I miss something (missing something is my passion!  🫡 )

**Comment by virzak** [comment]
Tested and it works great!
